omg/entities/elements.py

```diff
@@ -0,0 +1,35 @@
+import os
+import json
+from typing import Dict
+
+ELEMENTS_JSON_DIR = os.path.join(os.path.dirname(__file__), "Elements.JSON")
+
+
+class DotDict(dict):
```

Comment:
I appreciate the effort but I did not understand what the added value of DotDict is, instead of a normal dict. Why we cannot use `Elements.get("Fire")` instead of `Elements.FIRE`?